wiki/Tournaments/Chinese mania 4K Tournament 2017/zh.md

```diff
@@ -0,0 +1,208 @@
+[Shiki – Endless Dream (pompom’s Another)](https://osu.ppy.sh/b/601737)  
+[sakuzyo – Refel (Liberation)](https://osu.ppy.sh/b/1178870)  
+[-45 – EXILE (Anotehr_SideA)](https://osu.ppy.sh/b/815021)  
+[seleP – Scarlet Rose (Scarlet Rose)](https://osu.ppy.sh/b/1228278)  
+[Polyphony – Adrift (Marine)](https://osu.ppy.sh/b/658126)  
+[LeaF – I (_UJ’s Another)](https://osu.ppy.sh/b/928341)  
+[DJ Sharpnel – Cyber Induction （IcyWorld）](https://osu.ppy.sh/b/1021178)  
+[Helblinde – Rewrite Nightmare (Experiment)](https://osu.ppy.sh/b/719525)  
+[Helblinde – Memoria (Original Mix) (LeiN-‘s SC)](https://osu.ppy.sh/b/1101044)  
+[DET & Silantra – Luv Song (<3)](https://osu.ppy.sh/b/602841)  
+[Kobaryo – Pumpin’ Junkies (Kobaryo’s FTN-Remix) (Arcwin’s Maxima)](https://osu.ppy.sh/b/644265)  
+[A-Teens – Gimme! Gimme! Gimme! (Wafles’s Beginner 24)](https://osu.ppy.sh/b/1010130)  
+[senya – Zenaku no Itadaki ni Aru Shinjitsu ( EXH （LN))](https://osu.ppy.sh/b/1050840)  
+[96Neko – Paintings? Oh, yeah. (PaiLNfull)](https://osu.ppy.sh/b/1103346)  
+[LeaF – 4th smile (Prosperity)](https://osu.ppy.sh/b/1247978)  
+[Da Tweekaz – Wodka (short Ver.) (4K MX(Hard SV Edit.))](https://osu.ppy.sh/b/800952)
+
+TB:  
+[Camellia – Lunatic Rough Party!! (Long Ver.)（Lunatic）](https://osu.ppy.sh/b/646319)
+
+## 比赛回顾
+
+## 规则集
+
+### 赛事规则 
+
+1. **2017年华人osu!mania 4K大赛**(以下均简写为**CMT 4K 2017**)为华人区个人比赛，是由CMT团队举办的第二届华人osu!mania 4K大赛。比赛所用地图均为osu!mania 4K专用地图(4K专谱)。
+2. 每场比赛的地图库将于比赛前一周的周末由地图库管理员放出，图库内包含若干张Free-Mod地图:  
+	&emsp;a) 其中的Tie-breaker（决胜）地图仅用于决胜局比赛。  
+	&emsp;b) 所有地图均在Free-mod池下。
+3. 比赛日程将由组委会讨论公布。(组委会名单见公告)
+4. 若工作人员与统计员均不能就位，比赛将会被推迟。
+5. 比赛中如出现 Failed 情况，则视为负手。
+6. 选手可以使用自己的皮肤与视频设置。
+7. 如果出现平局情况，则该场比赛视作无效。
+8. 当选手掉线时，如果能够继续完成比赛，则应当在比赛结束后(使用 Shift+F12)立刻上传分数界面截图并提供网址。
+9. 如果选手因为掉线直接跳出 mp 房间界面而未能完成该图，则按以下方法处理：  
+	&emsp;a) 在比赛开始 30 秒内发生的，应予以重赛。  
+	&emsp;b) 否则视作失败（Failed）。
+10. 除非某张地图的比赛结果被列为无效，否则任何地图不能两次被用于同场比赛。
+11. 如果比赛选手因为特殊原因无法出席，最多允许等待 10 分钟，十分钟后做视为弃权。
+12. 选手自行确保比赛游戏过程中其游戏与操作系统的流畅性，例如以下情况不能重赛（包括但不限于）：  
+	- 网络原因导致的延迟、卡顿现象。   
+	- 系统弹出其他应用的通知或窗口。  
+	- 硬件故障。  
+	- 其他原因。
+13. 小组赛阶段某选手弃权时，当场比赛视作 “4:0 +1.0 分差比” 失败。
+14. 任何列出情况之外的意外将交由组委会讨论处理。
+15. 任何规则的修改都将被公示。
+16. 组委会保留公示内容优先使用简体中文的权利。
+17. 所有公示内容请在音游网查看。
+
+## 报名注册
+
+1. 每位参赛者应当独立报名(进入报名网站填写相关信息报名，网站地址将在报名开始时公布)。
+2. 玩家满足以下条件的可以报名参加CMT 4K 2017：  
+	- 玩家注册地区为中国（包括香港、澳门）或台湾（也可以将国籍在此范围内作为注册条件）。  
+	- 玩家赛时使用时区无限制。  
+	- 玩家使用汉语作为主要语言。  
+	- 本次比赛对rank无强制要求，建议4000rank之内的玩家参赛。  
+	- 如果参赛者在过去一年内于 osu!社区或在比赛中有过不良记录，我们可能会取消其参赛资格。  
+3. 报名成功的选手将收到确认邮件，同时也将被加入公示名单。
+4. 图池管理和比赛经理不得参赛。
+
+## 比赛赛程
+
+1. 在资格赛阶段，全部选手随机分为8人一组进行Multiplayer，每张图按照图池的顺序打2遍，并取最高分数。4张图分数之和就是选手的总成绩。
+2. 在小组赛阶段，32名选手将按照资格赛的成绩分为8组(小组赛名额可能随报名人数变化)。
+3. 每个小组采用循环积分赛制，两两均会比赛。
+4. 小组内每个选手的排名根据以下优先级排序：  
+	- 选手赢得更多场次。
+	- 选手有更高的{(赢图数)-(负图数)}。  
+	- 选手有更高的赢图数。  
+	- 选手有更高的 Σ{(分差)/(该图分数)}。  
+	- 以上皆平手时加赛的胜利者。
+5. 每个小组前__2__名选手将晋级到淘汰赛(人数可能根据实际报名人数变化)。
+6. 接下来的赛事均为双败赛制，即胜者进入下一阶段而败者进入败者组。
+7. 根据赛图安排，这些赛事的编排如下：  
+| 赛程 | 比赛编号 |
```

Comment:
Add an empty newline space here to separate the text and the table for GFM parsing to work.